src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { FC } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink/SearchLink';
+
+interface PeopleFiltersPerops {
```

Comment:
Typo in the interface name, it should be 'PeopleFiltersProps' instead of 'PeopleFiltersPerops'